chromium_src/content/browser/browser_context.cc

```diff
@@ -37,9 +40,17 @@ scoped_refptr<content::SessionStorageNamespace> CreateSessionStorageNamespace(
 }
 
 std::string GetSessionStorageNamespaceId(WebContents* web_contents) {
-  return web_contents->GetRenderViewHost()
-      ->GetDelegate()
-      ->GetSessionStorageNamespace(web_contents->GetSiteInstance())
+  RenderViewHostImpl* rvh_impl =
```

Comment:
mb we can save a couple of lines by getting `NavigationController` from webcontents and downcasting. Also `RenderViewHostImpl` is sort of deprecated and may disappear in the future

Reply:
Fixed in https://github.com/brave/brave-core/commit/3fc8805e41721096fcd14f06a2627e77b6743fc6